content/guides/tina-cloud/starter/local.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Run Tina Cloud Starter Locally
+last_edited: '2021-05-10T10:00:00.000Z'
+---
+
+## Fork the repository
+
+⚠️⚠️ Start by [**forking** the repositorty](https://github.com/tinacms/tina-cloud-starter) and then pull it down to your computer. ⚠️⚠️
+
+## Install
+
+> ℹ️ This project uses `yarn` as a package manager, if `yarn` isn't installed on your machine, open a terminal and run `npm install -g yarn`
+
+Install the project's dependencies:
+
+```
+yarn install
+```
+
+> ⚠️ If you'd like to use `npm` beware that there is no `package-lock.json` so we can't guarantee the dependencies are the same for you.
+
+## Run the project locally
+
+Copy the sample `.env.local.sample` file and run the local development server:
+
+```
+cp .env.local.sample .env.local
+```
+
+```
+yarn dev
+```
+
+> In your .env.local file `NEXT_PUBLIC_USE_LOCAL_CLIENT` should be set to `1`, other values can be ignored for now.
+
+This command starts the GraphQL server and the Next.js application in development mode. It also regenerates your schema types for TypeScript and GraphQL so changes to your `.tina` config are reflected immediately.
+
+One of the most interesting aspects of the Tina Cloud Content API is that it doesn't actually require anything from the Cloud to work locally. Since Tina is by default a Git-backed CMS, everything can be run from your local filesystem via the CLI. :sunglasses:
```

Comment:
```suggestion
One of the most interesting aspects of the Tina Cloud Content API is that it doesn't actually require anything from the Cloud to work locally. Since Tina is by default a Git-backed CMS, everything can be run from your local filesystem via the CLI. 😎 
```